apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/SpanConfiguration.java

```diff
@@ -57,6 +57,16 @@
         .dynamic(true)
         .buildWithDefault(TimeDuration.of("5ms"));
 
+    private final ConfigurationOption<TimeDuration> exitSpanMinDuration = TimeDurationValueConverter.durationOption("ms")
```

Comment:
```suggestion
    private final ConfigurationOption<TimeDuration> exitSpanMinDuration = TimeDurationValueConverter.fineDurationOption("ms")
```

Reply:
Hm, I thought I had done that already

Reply:
Ah, I only did the `buildWithDefault`. Sorry!